### DISCUSSION TITLE
part-ii/2018-115/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#47] [Question comments]
Part II 2018 Principles of Quantum Mechanics: Paper 2, Section II, D 

https://questions.tripos.org/part-ii/2018-115/

**Comment by silkca** [comment]
The first half is a calculation described in lectures.

The second half uses the same method as the first, but relies on some information about the eigenstates of $J_{z}$, in particular their form in terms of exponentials and Legendre polynomials, to determine angular momentum values.
